firebase_admin/auth.py

```diff
@@ -94,10 +96,653 @@ def verify_id_token(id_token, app=None):
       ValueError: If the JWT was found to be invalid, or if the App was not
           initialized with a credentials.Certificate.
     """
-    token_generator = _get_token_generator(app)
+    token_generator = _get_auth_service(app).token_generator
     return token_generator.verify_id_token(id_token)
 
 
+def get_user(uid, app=None):
+    """Gets the user data corresponding to the specified user ID.
+
+    Args:
+        uid: A user ID string.
+        app: An App instance (optional).
```

Comment:
If an app is not explicitly provided (which is the more common use case), we use the default app returned by `firebase_admin.get_app()`. This is a feature common to all APIs in this library. Almost all public functions accept an optional `app` argument. 

The equivalent logic in Java/Android SDKs is:
```
FirebaseAuth.getInstance() // Uses default app
FirebaseAuth.getInstance(app) // Uses the provided app
```